docs/other-topics/upgrade.md

```diff
@@ -315,6 +315,15 @@ User.belongsToMany(Country, {
 - A few bugs in how indexes were named have been fixed. This means your index names could change.
 
 ## Minor Breaking changes
+### Renamed APIs
+- `Sequelize.QueryInterface` has been renamed to `Sequelize.AbstractQueryInterface`. Previously, you could do: 
+```javascript
+const { QueryInterface } = require('@sequelize/core');
+```
+The new import would be:
+```javscript
+const { AbstractQueryInterface } = require('@sequelize/core');
+```
```

Comment:
I don't know if this example is necessary, considering there is a good number of APIs that have been renamed (many of which are yet to be documented), but if you do add it:

- prefer `import` to `require`
- added a bit of spacing
- I used `sequelize` for the old package, since that's what v6 uses

````suggestion

```javascript
import { QueryInterface } from 'sequelize';
```

The new import would be:

```javascript
import { AbstractQueryInterface } from '@sequelize/core';
```
````

Reply:
Sure - I'll remove the example